### PR TITLE
chore: add issue templates

### PR DESCRIPTION
**Pull request by pinussilvestrus**
They were missing after moving the repo to `camunda`.

### CODE REVIEW

**Comment by Skaiir** [comment]
Couldn't we use [issue forms](https://github.com/camunda/camunda-modeler/blob/develop/.github/ISSUE_TEMPLATE/BUG_REPORT.yml) here instead? Or are we just trialing those out on `camunda-modeler`?

Feel free to merge this if that's the case then, 👍

**Comment by pinussilvestrus** [comment]
Good catch, I don't know whether we already want to use the forms approach in a wider manner.

/cc @MaxTru

**Comment by MaxTru** [comment]
> Good catch, I don't know whether we already want to use the forms approach in a wider manner.
> 
> /cc @MaxTru

Let's make this decision as a team (perhaps in the next Weekly). I can put something up.

Until then, I would keep this PR as is and merge. Migrating should be straight forward once we decided.

**Review by MaxTru — APPROVED** [review]

**Pull request merged**
